app/views/content/subjects/maths/_article.html.erb

```diff
@@ -1,185 +1,160 @@
-      are more likely to be in employment than those who do not master these skills.
+      If you train to teach maths, you could also be eligible to receive a scholarship or bursary of up to £29,000.
     </p>
   </section>
 
   <section class="col col-space-s col-content-max">
     <%= render TextBlockWithTwoImagesComponent.new(
-      heading: "Maths is for everyone",
+      heading: "Making maths add up",
       colour: "pink",
       image_1_path: "static/content/hero-images/0017.jpg",
       image_1_alt: "A teacher sitting with a student helping them with a maths problem",
       image_2_path: "static/content/hero-images/0024.jpg",
       image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
     ) do %>
       <p>
-        If maths is your passion, you know that it is not just learning formulas or
-        equations. It is a way of thinking, and problem solving, that’s important
-        in every walk of life.
+        Maths is critical to science, technology and engineering as well as being fundamental for most jobs.
       </p>
 
       <p>
-        An exciting career spent passing your passion on to the next generation
-        could be the career for you.
+        It is our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.
+      </p>
+      <p>
+        A good understanding of maths offers big benefits for young people as well as helping towards a strong economy.
       </p>
     <% end %>
   </section>
 
-  <section class="col col-640 col-space-s">
-    <%= render Content::QuoteComponent.new(
-      text: "You can literally change young people’s lives by being a maths teacher.
-          Be the person who helps them see the beauty and importance of maths.
-          It is the best job ever!",
-      name: "Kevin, maths teacher in Sheffield",
-      background: "white",
-      large: true
-    ) %>
-  </section>
-
   <section class="col col-720 col-horizontal col-space-m">
     <div>
-      <h2 class="heading-s">What you'll be teaching</h2>
-
-      <p>
-        In England, there are important aspects of maths to teach each age
-        group. As a secondary school teacher, maths education means teaching
-        pupils:
-      </p>
-    </div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the content of the national curriculum. The themes you are likely to cover include: </p>
+      <p>11 to 14 year olds (key stage 3):</p>
+      <ul class="vertical-tags">
+        <li class="tag">Number</li>
+        <li class="tag">Algebra</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
+        <li class="tag">Geometry and measures</li>
+        <li class="tag">Probability and statistics</li>
+      </ul>
 
-    <div>
+      <p>14 to 16 year olds (key stage 4):</p>
       <ul class="vertical-tags">
-        <li class="tag">Numbers</li>
+        <li class="tag">Number</li>
```

Comment:
```suggestion
        <li class="tag">Numbers</li>
```
worth combining numbers and algebra on the same line to take up less space?

Reply:
I think they are quite different topics though?